tiatoolbox/tools/pyramid.py

```diff
@@ -481,6 +481,12 @@ def __init__(
         if renderer is None:
             renderer = AnnotationRenderer()
         self.renderer = renderer
+        self.overlap = int(1.5 * renderer.blur_radius)
```

Comment:
Where does the magic number 1.5 come from?

Reply:
Can we make this a named constant or add a comment to explain why this number is here?

Reply:
added a comment to explain the 1.5